<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YAGB</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="outerLayout">
    <item>
     <widget class="QWidget" name="topPane" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="topPaneLayout">
       <item>
        <widget class="LcdWidget" name="lcdWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>288</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>320</width>
           <height>288</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="topPaneColumn1Container" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="topPaneColumn1Layout">
          <item>
           <widget class="QGroupBox" name="cpuRegsGroupbox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>CPU</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="cpuRegsAfLabel">
                 <property name="text">
                  <string>AF</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="HexTextField4" name="cpuRegsAf"/>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="cpuRegsBcLabel">
                 <property name="text">
                  <string>BC</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="HexTextField4" name="cpuRegsBc"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="cpuRegsDeLabel">
                 <property name="text">
                  <string>DE</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="HexTextField4" name="cpuRegsDe"/>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="cpuRegsHlLabel">
                 <property name="text">
                  <string>HL</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="HexTextField4" name="cpuRegsHl"/>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="cpuRegsSpLabel">
                 <property name="text">
                  <string>SP</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="HexTextField4" name="cpuRegsSp"/>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="cpuRegsPcLabel">
                 <property name="text">
                  <string>PC</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="HexTextField4" name="cpuRegsPc"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="OneCharacterButton" name="cpuFlagZ">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Z</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="OneCharacterButton" name="cpuFlagC">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>N</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="OneCharacterButton" name="cpuFlagH">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>H</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="OneCharacterButton" name="cpuFlagN">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>C</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="OneCharacterButton" name="cpuFlagIrqs">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>IRQ</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="irqsGroupbox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>IRQ</string>
            </property>
            <layout class="QGridLayout" name="irqsFormLayout"/>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="lcdRegsGroupbox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>LCD</string>
         </property>
         <layout class="QFormLayout" name="lcdRegsFormLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Tile Bitmaps</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="LcdWidget" name="patternViewerLcdWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>271</width>
              <height>407</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>271</width>
              <height>407</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Tile Map</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="LcdWidget" name="tileMapViewerLcdWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>543</width>
              <height>543</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>543</width>
              <height>543</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="logTextarea"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LcdWidget</class>
   <extends>QWidget</extends>
   <header>gui/LcdWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HexTextField4</class>
   <extends>QLineEdit</extends>
   <header>gui/HexTextField.hpp</header>
  </customwidget>
  <customwidget>
   <class>OneCharacterButton</class>
   <extends>QToolButton</extends>
   <header>gui/OneCharacterButton.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
